homeassistant/components/home_connect/strings.json

```diff
@@ -22,6 +22,9 @@
     }
   },
   "exceptions": {
+    "auth_error": {
```

Comment:
Wouldn't it be better to implement a reauth flow than asking the user to delete and readd the integration?

Reply:
Sure, in the PR description I mentioned that I tried, but after completing the reauth, a new config entry is created

Reply:
I think it worth trying to fix that than asking the user to delete and readd the integration

Reply:
Sure it is, I posted the PR like this with the idea of asking for help. 
Do you have any suggestion about how to reconfigure the config entry based on its own ID?

Reply:
If there's no unique id we should only allow a single config entry.

Reply:
Here's how to only allow a single config entry:

https://developers.home-assistant.io/docs/creating_integration_manifest/#single-config-entry-only

I think a single config entry should be enough. It's hard to imagine when you would need to set up more than one user account in the same home.